aws/data_source_aws_lb_target_group.go

```diff
@@ -40,6 +40,11 @@ func dataSourceAwsLbTargetGroup() *schema.Resource {
 				Computed: true,
 			},
 
+			"protocol_version": {
```

Comment:
Nice catch! I've heard from a friend that it's really bad to add an attribute to the resource but not the data source.

Reply:
Thanks! I almost missed it.